apps/backend/src/collectivites/fixtures/collectivites.fixtures.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Collectivite,
+  collectiviteBucketTable,
+  collectiviteTable,
+  CollectiviteType,
+  collectiviteTypeEnum,
+} from '@/backend/collectivites/index-domain';
+import { addTestUser, TestUserArgs } from '@/backend/users/index-domain';
+import {
+  DatabaseServiceDto,
+  getErrorMessage,
+} from '@/backend/utils/index-domain';
+import { eq } from 'drizzle-orm';
+
+export type TestCollectiviteArgs = {
+  nom?: string;
+  type?: CollectiviteType;
+};
+
+// ajoute une collectivité
+export async function addTestCollectivite(
+  databaseService: DatabaseServiceDto,
+  { nom, type }: TestCollectiviteArgs
+) {
+  try {
+    const result = (await databaseService.db
+      .insert(collectiviteTable)
+      .values([
+        {
+          nom:
+            nom || `Collectivité ${Math.random().toString().substring(2, 6)}`,
+          type: type || collectiviteTypeEnum.EPCI,
+        },
+      ])
+      .returning()
+      .then(([c]) => c)) as Collectivite;
```

Comment:
```suggestion
      .then(([c]) => c));
```

Ces cast sont "dangereux" car ils peuvent cacher des sujets entre la DB et nos types dans le domaine. Idéalement, drizzle fait le taff et si conflit il y a entre le returnType de drizlle et `Collectivite`, c'est que l'on a un sujet

Reply:
Ok fait le map comme discuté pour gérer les enums